### PR TITLE
Bios

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

**Review by mimiflynn — COMMENTED** [review]
I just realized I was removed from the review, but I have this one thing.

**Comment by robmoffat** [comment]
please don't merge @psmulovics - there may be other people wanting to add their bios

**Comment by robmoffat** [comment]
Bios on this page: https://deploy-preview-229--open-source-readiness.netlify.app/docs/about

**Comment by psmulovics** [comment]
We can mark it draft or add DO NOT MERGE in the subject to prevent that. I know draft PRs don't get the preview, so let's do the subject

**Comment by robmoffat** [comment]
@gravax

**Comment by robmoffat** [comment]
@Neetuj done

**Review by gravax — APPROVED** [review]
Works for me.

File: docs/bok/Contribution.md [hunk 1]
```diff
@@ -44,30 +65,37 @@ Cara Delia advocates open source principles and practices by contributing to ext
...
 Peter Smulovics is an Executive Director of Enterprise Application Infrastructure focusing on Microsoft technologies at Morgan Stanley. He is a publicist for IDG and for Architecture Journal; author and co-author of several books and a returning presenter on TechEd, DevDays, Techready, Agile DevCon, TechNet. He is a founder member of AgileAlliance, Architecture Forum, Architect Academy and .NET forum, Technical Lead on several OSS projects like Novell/JBoss’ Hibernate, Apache Velocity and similar. He is Microsoft, Ariba, JBoss, and SiteCore certified and also a trainer on several technologies. Working in IT since 1995, has project experience in different fields (FMCG, public, financial, energy, telecom, pharmaceutical, mobile, music, security). He worked at Compaq as Oracle DBA, worked at Microsoft on C#, .NET, Dynamics, Visual Studio .NET, SQL Server, Office, ASP.NET, Entity Framework, and Azure, and is the main organizer for the FSI Autism Hackathon.
 </Bio>
 
+<Bio name="Thomas Cooper" organisation="RBC" image="/img/people/thomas-cooper.jpeg">
+Director, Open Source Software Development
+</Bio>
```
Comment (coopernetes) [inline docs/bok/Contribution.md:74]:
```suggestion
<Bio name="Thomas Cooper" organisation="RBC" image="/img/people/thomas-cooper.jpeg">
Thomas Cooper is the Director, OSPO and part of the Developer Advocacy team at Royal Bank of Canada. He began his IT career in network operations at IBM in 2011 and joined RBC in 2013. With over a decade of industry experience in the financial sector, he has been a platform engineering lead, application developer, systems architect and community leader & champion for open source, InnerSource, best practices and cloud native technologies. In addition to his technical expertise, he has lead various community efforts for improving developer experience & productivity at RBC. He is passionate about improving the lives of others, software quality & craftsmanship and building tools that are a joy to use. 
</Bio>
```

**Comment by robmoffat** [comment]
can we merge this?

File: website/src/theme/Bio/styles.module.css
Comment (mimiflynn) [inline website/src/theme/Bio/styles.module.css:1]: Spacing mismatches.

**Review by mimiflynn — COMMENTED** [review]
My only issue is css spacing.

File: website/src/theme/Bio/styles.module.css
Comment (psmulovics) [reply website/src/theme/Bio/styles.module.css:1]: Would there be a linter you would suggest that could do both CSS, HTML, MD, etc that we could be running as part of the build and would show us these issues, @mimiflynn ?

Comment (mimiflynn) [reply website/src/theme/Bio/styles.module.css:1]: Of course, in addition, I use VSCode to format files on save according to those settings. Most issues can be fixed automatically.

Comment (robmoffat) [reply website/src/theme/Bio/styles.module.css:1]: fixed

**Review by mimiflynn — APPROVED** [review]

**Review by mimiflynn — APPROVED** [review]

**Pull request merged**